### PR TITLE
Updated invite user UI, removed readonly email field, #PG-3048

### PR DESCRIPTION
**Pull request by AltamashShaikh**
### Description:

Updated invite user UI, removed readonly email field
Fixes: #PG-3048.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Comment by sgiehl** [comment]
@AltamashShaikh As far as I know this should only be changed for the cloud onboarding. Thus this change here won't be needed.
And a general note: Setting the email/login field as hidden, might break password managers from picking it up, which can be avoided by hiding the input using css.

**Comment by AltamashShaikh** [comment]
> @AltamashShaikh As far as I know this should only be changed for the cloud onboarding. Thus this change here won't be needed. And a general note: Setting the email/login field as hidden, might break password managers from picking it up, which can be avoided by hiding the input using css.

Taking forward to what we discussed on Slack
1. Make the label changes in core
2. Hide the email field via css in Cloud plugin

File: plugins/UsersManager/API.php [hunk 1]
```diff
@@ -1612,7 +1612,7 @@ public function generateInviteLink($userLogin, $expiryInDays = 7, $passwordConfi
         Piwik::postEvent('UsersManager.inviteUser.generateInviteLinkToken', [$userLogin, $user['email']]);
 
         return SettingsPiwik::getPiwikUrl().'index.php?'.Url::getQueryStringFromParameters([
-                'module' => 'Login',
+                'module' => Piwik::getLoginPluginName(),
```
Comment (AltamashShaikh) [inline plugins/UsersManager/API.php:1615]: This fixes the issue of wrong plugin being used while generating the copy invite user link

File: plugins/Login/templates/invitation.twig [hunk 2]
```diff
@@ -22,25 +23,25 @@
...
                 <input type="hidden" name="token" value="{{ token }}"/>
                 <div class="row">
-                    <div class="col l10 s12 input-field">
+                    <div class="col l10 s12 input-field" id="invite-user-email-div">
                         <input type="text" name="email" id="email" value="{{ user.email }}" size="20" readonly="readonly"
                                tabindex="0"/>
                         <label for="email">{{ 'UsersManager_Email'|translate }}</label>
                     </div>
-                    <div class="col l10 s12 input-field">
+                    <div class="col l10 s12 input-field invite-user-pwd-div">
```
Comment (sgiehl) [inline plugins/Login/templates/invitation.twig:31]: those should be ids, right?

**Review by sgiehl — CHANGES_REQUESTED** [review]

File: plugins/Login/templates/invitation.twig (hunk 2 above)
Comment (sgiehl) [reply plugins/Login/templates/invitation.twig:31]: Ah. doesn't make sense. It's there twice...

**Review by sgiehl — APPROVED** [review]

**Pull request merged**